src/AdapterManager.js

```diff
@@ -100,48 +103,51 @@ var adapterManager = function (options) {
         }
         _pluginCallback = callback;
 
-        var checkPluginEnabledForAdapter = function (obj, callback) {
-            var endTime = Date.now() + obj.timeout;
+        var checkPluginEnabledForAdapter = function (adpterResponse, callback) {
+            var endTime = Date.now() + (adpterResponse.timeout >= 1000 && adpterResponse.timeout <= 10000 ? adpterResponse.timeout : 5000);
```

Comment:
I'm glad that you added the timeout range check here. We should make sure to mention to @vpitts that these range min/max values need to be documented as part of the API docs, so that publishers are aware of the constraints.